google-cloud-spanner/pom.xml

```diff
@@ -365,6 +366,46 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>slow-tests</id>
+      <build>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-surefire-plugin</artifactId>
+              <executions>
+                <execution>
+                  <id>default-test</id>
+                  <configuration>
+                    <groups>com.google.cloud.spanner.categories.SlowTest</groups>
+                  </configuration>
+                </execution>
+              </executions>
+            </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-failsafe-plugin</artifactId>
+              <executions>
+                <execution>
+                  <id>default</id>
+                  <configuration>
+                    <groups>com.google.cloud.spanner.categories.SlowTest</groups>
+                  </configuration>
+                </execution>
+                <execution>
+                  <id>parallel-integration-test</id>
```

Comment:
What is the intention with this execution? The name seems to indicate that it will execute parallel integration tests, but if I understand the configuration correctly, it will just skip all tests. Could we maybe add a comment to this section to explain why it's here?

Reply:
This is because the default configuration (line 88-98) defines an execution for the integration-test goal with such id. Thus, we need to override it in the slow tests profile to NOOP, otherwise the tests with the category `ParallelIntegrationTest` will be executed (as per the default one). I added a comment in this section.